wikipages/II. Project Layout.md

```diff
@@ -0,0 +1,110 @@
+A typical project will have at least the following resources:
```

Comment:
"typical" is difficult to define here. I gave this a bit of thought - I like what's here, though what do you think about including an ml-config/rest-api.json file?